src/Resources/translations/validators.fr.yaml

```diff
@@ -0,0 +1,4 @@
+monsieurbiz_cms_page:
+    ui:
+        slug:
+            unique: 'Ce slug est déjà utilisé pour une autre page.'
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏Improve translation